source/module_elecstate/elecstate.h

```diff
@@ -0,0 +1,31 @@
+#ifndef ELECSTATE_H
+#define ELECSTATE_H
+
+#include "module_psi/psi.h"
+#include "module_hamilt/matrixblock.h"
+#include "src_pw/charge.h"
+
+namespace ModuleElecS
+{
+
+class ElecState
```

Comment:
namespace ModuleElecS, class ElecState, module_elecstate, elecstate.h, the inconsistency here may confuse developers.